config/rbac/role.yaml

```diff
@@ -13,6 +13,18 @@ rules:
   - list
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
```

Comment:
we shouldn't need pods anymore

Reply:
As you've seen in your next comment [we still list pods here](https://github.com/p-strusiewiczsurmacki-mobica/das-schiff-network-operator/blob/extended-routing-visability/pkg/monitoring/endpoint.go#L326)